Dockerfile.docs

```diff
@@ -70,8 +56,7 @@ USER node
 HEALTHCHECK --interval=5s --timeout=30s --retries=3 \
   CMD node /usr/src/app/health-check.js
 
-ENTRYPOINT ["/usr/local/bin/npm", "run"]
-CMD ["start"]
+CMD ["yarn", "start"]
```

Comment:
Read an interesting bit about using yarn vs node in the CMD of dockerfiles today.

See num 2: https://dev.to/nodepractices/docker-best-practices-with-node-js-4ln4

Reply:
👀